app/Repositories/PhotoRepository.php

```diff
@@ -42,10 +42,17 @@ public function __construct(AWS $aws, Registrar $registrar)
      */
     public function create(array $data, $signupId)
     {
+        // Include signup_id when we create the Event
+        $data['signup_id'] = $signupId;
+
         // Don't send quantity and why_participated - we don't want this to live on the post_photo event.
         $postEvent = Event::create(array_except($data, ['quantity', 'why_participated']));
 
-        $fileUrl = $this->aws->storeImage($data['file'], $signupId);
+        if (isset($data['file'])) {
+            $fileUrl = $this->aws->storeImage($data['file'], $signupId);
+        } else {
+            $fileUrl = 'default';
+        }
 
         $editedImage = $this->crop($data, $signupId);
```

Comment:
@katiecrane this is a dumb question but just want to double check! If `$data['file']` is not set, is it possible it'll have crop values? Just want to make sure for when it goes to the [`crop`](edit_image) function that this doesn't break either.